test/karma/package.json

```diff
@@ -27,7 +27,7 @@
   },
   "devDependencies": {
     "@stencil/sass": "^3.0.0",
-    "@types/node": "^16.18.36",
```

Comment:
Ugh - looks like the Renovate config I added for Karma didn't take properly :-( Blocking until we can fix that